cmd/placement/config.go

```diff
@@ -76,6 +80,7 @@ func newConfig() *config {
 	flag.IntVar(&cfg.healthzPort, "healthz-port", cfg.healthzPort, "sets the HTTP port for the healthz server")
 	flag.StringVar(&cfg.certChainPath, "certchain", cfg.certChainPath, "Path to the credentials directory holding the cert chain")
 	flag.BoolVar(&cfg.tlsEnabled, "tls-enabled", cfg.tlsEnabled, "Should TLS be enabled for the placement gRPC server")
+	flag.BoolVar(&cfg.metadataEnabled, "metadata-enable", cfg.metadataEnabled, "Expose the placement tables, using the same HTTP port as healthz")
```

Comment:
nit: metadata-enabled

Reply:
Sorry for the delay. Done, also resolved the conflict.